decidim-accountability/lib/decidim/accountability/list_engine.rb

```diff
@@ -4,18 +4,22 @@
 require "kaminari"
 
 module Decidim
-  module Results
-    # This is the engine that runs on the public interface of `decidim-results`.
+  module Accountability
+    # This is the engine that runs on the public interface of `decidim-accountability`.
     # It mostly handles rendering the created results associated to a participatory
     # process.
-    class Engine < ::Rails::Engine
-      isolate_namespace Decidim::Results
+    class ListEngine < ::Rails::Engine
```

Comment:
`Engine` is our standard name for the frontend engine of a feature. I think we can keep it.